### ISSUE TITLE
InfluxDBClientError: partial write: field type conflict: input field [...] is type float, already exists as type integer

### ISSUE THREAD

**Issue by amotl** [daq-tools/ttnlogger#1]
We received this problem report after adding a new sensor transmitting the `pressure` measurement:
```
('ERROR:', InfluxDBClientError(u'400: {"error":"partial write: field type conflict: input field \"pressure\" on measurement \"data\" is type float, already exists as type integer dropped=1"}\n',))
```

The error message is
> InfluxDBClientError: partial write: field type conflict: input field "pressure" on measurement "data" is type float, already exists as type integer

**Comment by amotl** [comment]
## Introduction
We know this from [Kotori](https://github.com/daq-tools/kotori) already and described it extensively at [Data acquisition for specific measurement fields gets rejected](https://community.hiveeyes.org/t/data-acquisition-for-specific-measurement-fields-gets-rejected/897).

## Solution
As there is no way to amend the data format of the `pressure` field already stored in InfluxDB, the quickest solution would be to just drop the database using

    influx -execute 'DROP DATABASE ttnlogger'

after assuring your new telemetry values are actually being submitted as `float` values.

## Outlook
We should apply the same logic here we are using in Kotori per [`convert_floats(data)`](https://github.com/daq-tools/kotori/blob/ff2ba2fd/kotori/io/protocol/util.py#L88-L111) already: Convert anything to float which eventually looks numeric.

It's usually perfectly [DWIM](https://en.wikipedia.org/wiki/DWIM) for our use case of transmitting telemetry values and probably reasonably sane to assume them as floating point numbers. However I admit there might by cases challenging that as a default behavior.

**Comment by amotl** [comment]
Should be implemented per 28bb699. However, I don't own an appropriate device to test this, yet.

**Comment by amotl** [comment]
Closing this in the hope it will be fine. @thiasB is operating an instance of `ttnlogger` and hasn't complained about this detail yet
